internal/bootstrap/bootstrap.go

```diff
@@ -85,14 +86,29 @@ func (i *initializer) Run(ctx context.Context) (*embed.Config, error) {
 
 // ChangeFilePermissions changes the file permissions of all files in the given directory and its subdirectories recursively.
 func ChangeFilePermissions(dir string, mode os.FileMode) error {
+	info, err := os.Stat(dir)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return nil
+		}
+		return fmt.Errorf("error stating directory %s: %v", dir, err)
```

Comment:
```suggestion
		return fmt.Errorf("error stating directory %s: %w", dir, err)
```
Wrapping the returned error seems more appropriate here. Also because this returned error is wrapped with another error in the caller, so it may become inconsistent if we wrap one but not other in the tree.